### ISSUE TITLE
Show see the full error when building my App using webpack with the webpackbar plugin?

### ISSUE THREAD

**Issue by Flcwl** [nuxt/webpackbar#72]
### What problem does this feature solve?

Can't see the full error message when building My App. It would be a problem if error when building, I can't troubleshoot errors quickly maybe running webpack compiler in CI or Docker. 

### What does the proposed changes look like?

Now It is pretty good and useful for me, But  I really hope it can show the full error when an error occurs in the future, It will be better.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/webpackbar/issues/c46">#c46</a>)</em></sub></div>

**Comment by scrptd95** [comment]
Yeah, would also like to have a complete error log. Currently it's difficult to tell why a compilation fails. Or am i missing a config options ?
